schemas/jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is there any reason you included jsconfg.json and tsconfig.json in the schema folder?

Reply:
I hope it's clear from my other response.